src/server/middleware/api/plausible.js

```diff
@@ -0,0 +1,80 @@
+const { isProduction } = require('../../utils')
+const NodeCache = require('node-cache')
+require('isomorphic-fetch')
+
+const ScriptName = '/pl/script.js'
```

Comment:
Why are these variables capitalized? If they're constants, shouldn't they be completely uppercase 🤔

Reply:
That said, maybe these constants should be shared with the other files that require them? [DefaultSEO](https://github.com/iterative/dvc.org/pull/3423/files#diff-aa1a28d5dbb813758bec2678c00fcb7cf9104a2b5cfa9bbe8313270feaeec84a) could use this link.

Reply:
Thanks for pointing it out.